openapi/src/main/java/io/micronaut/openapi/swagger/core/jackson/mixin/OpenAPIMixin.java

```diff
@@ -36,6 +37,7 @@
 public abstract class OpenAPIMixin {
 
     @JsonAnyGetter
+    @JsonInclude
```

Comment:
is `@JsonInclude` necessary? Do we have any test to verify it is necessary?

Reply:
@sdelamo Yeas, it necessary. We don't need to test it. These classes just copied from swagger-core project. This change i got from here: https://github.com/swagger-api/swagger-core/pull/4468

I keep these classes up to date all the time. Why did I copy this piece from swagger-core? For two reasons:

1. This allowed me to remove a bunch of unnecessary dependencies from the project, i.e. from swagger-core, we used only the jackson settings.

2. Now there is a possibility to customize logic, due to this we managed to fix a couple of problems in serialization / deserialization.